docs/contribute/credits.rst

```diff
@@ -111,6 +111,7 @@ Contributors
 - Ville Skyttä <ville.skytta@iki.fi>
 - Wichert Akkerman <wichert@wiggy.net>
 - Howard Liao <howard940114@gmail.com>
+- Michael Njoku <mnjoku@umich.edu>`_
```

Comment:
Please use the GitHub URL for your username so that you don't disclose your email address to spam harvesters. You're welcome.

Also please sort your name alphabetically in the list.